src/xdp/bind.c

```diff
@@ -764,14 +764,19 @@ XdpIfSetInterfaceOffload(
     _In_ VOID *InterfaceOffloadHandle,
     _In_ XDP_INTERFACE_OFFLOAD_TYPE OffloadType,
     _In_ VOID *OffloadParams,
-    _In_ UINT32 OffloadParamsSize
+    _In_ UINT32 OffloadParamsSize,
+    _Out_writes_bytes_opt_(*OffloadResultWritten) VOID *OffloadResult,
+    _In_ UINT32 OffloadResultSize,
+    _Out_opt_ UINT32 *OffloadResultWritten
     )
 {
     XDP_INTERFACE_SET *IfSet = (XDP_INTERFACE_SET *)IfSetHandle;
 
+    #pragma warning(suppress:6001) // Using uninitialized memory '*OffloadResultWritten'
```

Comment:
Should we fix the annotation of `SetInterfaceOffload` instead of suppressing?

Reply:
I still don't understand why SAL is complaining here. It's an `_Out_opt_` parameter being unconditionally passed to another function that takes an `_Out_opt_` parameter. I couldn't make sense of this one.

Reply:
Weird...